src/chain_spec.rs

```diff
@@ -663,7 +663,7 @@ fn v1_testnet_genesis(
                 ),
             ],
             timelock: V1Time::MINUTES * 15,
-            bridge_limit: (25_000_000_000, V1Time::DAYS * 1),
+            bridge_limit: (30_000_000_000, V1Time::DAYS * 1),
```

Comment:
A change in the chain spec might cause a fork. It won't change anything in the live chain. It will likely just start a new chain with a different genesis.

Reply:
Yep - OK - I'll remove from `master` and leave in `develop`. I don't think it does cause a fork FWIW, possibly because we are not re-generating the chain_spec files and just using the original ones we launched with.

Reply:
Makes sense. If we don't regenerate the chain spec, this will be a no-op.

Reply:
I reverted this change anyway.